cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -92,6 +104,7 @@ func (n *node) isSoftOwnedBy(other *node) bool {
 type objectGraph struct {
 	proxy     Proxy
 	uidToNode map[types.UID]*node
+	crds      map[string]*crdInfo
```

Comment:
```suggestion
	types      map[string]*crdInfo
```
(same reason described above)